docs/src/app/components/pages/components/BottomSheet/Page.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Title from 'react-title-component';
+
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import bottomSheetReadmeText from './README';
+import BottomSheetSimpleExample from './ExampleSimple';
+import bottomSheetSimpleExampleCode from '!raw!./ExampleSimple';
+import BottomSheetCloseableExample from './ExampleCloseable';
+import bottomSheetCloseableExampleCode from '!raw!./ExampleCloseable';
+import BottomSheetInsetExample from './ExampleInset';
+import bottomSheetInsetExampleCode from '!raw!./ExampleInset';
+import BottomSheetModalExample from './ExampleModal';
+import bottomSheetModalExampleCode from '!raw!./ExampleModal';
+import bottomSheetCode from '!raw!material-ui/BottomSheet/BottomSheet';
+
+const descriptions = {
```

Comment:
Using this pattern is deprecated. We can simply add the description above the example.
For instance, have a look at https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/Dialog/ExampleAlert.js#L7.